libnmstate/netapplier.py

```diff
@@ -104,10 +103,11 @@ def _choose_checkpoint(dbuspath):
 
 def _apply_ifaces_state(desired_state, verify_change, commit,
                         rollback_timeout):
-    current_state = state.State({Constants.INTERFACES: netinfo.interfaces()})
+    current_state = state.State(netinfo.show())
```

Comment:
This seems to be an unrelated change. It would be better to do them separately in the future.

Reply:
This is related as `State` now holds the route configuration also.